### DISCUSSION TITLE
Installation Support for Orca Slicer

### DISCUSSION THREAD

**Discussion by zuverck** [jschuh/klipper-macros#142] [Ideas]
Any chance you could extend the setup instructions with instructions for Orca Slicer.  Specifically modifications and examples for Start G-code, End G-code, etc.  I have messed around with the stock configs and like some of the options that it gives around calibration scripts.  It is nice that the slicer is a fork of bamboo and as such would allow a single slicer for the Carbon and my other klipper printers utilizing your macros.  

https://github.com/SoftFever/OrcaSlicer/releases

**Comment by thalk84** [comment]
I use with Orca and have been for a couple month I think and have great results! Just use Prusa Slicer instructiuons.
